docs/hall_of_fame.rst

```diff
@@ -0,0 +1,26 @@
+Hall of Fame
+============
+Albumentations are widely used in Computer Vision Competitions at Kaggle an other platforms.
+
+Here are the links to the competitions, names of the winners and to their solutions.
+
+
+Kaggle
+======
+
+`Humpback Whale Identification <https://www.kaggle.com/c/humpback-whale-identification>`_
+-----------------------------------------------------------------------------------------
+
+5. `Roman Solovyov <https://www.kaggle.com/zfturbo>`__, `Weimin Wang <https://www.kaggle.com/weimin>`__
+    `blog post <https://weiminwang.blog/2019/03/01/whale-identification-5th-place-approach-using-siamese-networks-with-adversarial-training/>`__,
+    `code <https://github.com/aaxwaz/Humpback-whale-identification-challenge>`__
+
+`TGS Salt Identification Challenge <https://www.kaggle.com/c/tgs-salt-identification-challeng>`_
+------------------------------------------------------------------------------------------------
+1. `b.e.s. <https://www.kaggle.com/ybabakhin>`__, `phalanx <https://www.kaggle.com/phalanx>`__
+    `solution <https://www.kaggle.com/c/tgs-salt-identification-challenge/discussion/69291>`__,
+    `code <https://github.com/ybabakhin/kaggle_salt_bes_phalanx>`__,
+    `pre-print <https://arxiv.org/abs/1904.04445>`__
+
+Topcoder
+========
```

Comment:
not completed yet

Reply:
This is an example. It is incomplete. I am thinking about asking people to create PR's to add themselves to the list.